docs/index.html

```diff
@@ -0,0 +1,189 @@
+			<label><input type="radio" name="detection-type" value="text"> Text detection <span
+					class="label-description">Works best with high contrast...</span></label>
+			<div class="upload-label">Analyze image file:</div>
+			<label class="upload-button"><input name="detection-file" type="file" accept="image/*">Choose file</label>
+		</div>
+	</header>
+
+	<div id="imageDisplay">
+		<div class="info-text">
+			<p>
+				This is the shape detection API implementation demo
+			</p>
+			<p>
+				The following demo allows you to detect shapes, barcodes and faces from an uploaded image.
+			</p>
+			<p>
+				Refer to the
+				<a href="https://ckeditor.com/blog/How-to-detect-human-faces-and-other-shapes-in-JavaScript/"
+					target="_blank">
+					How to detect human faces (and other shapes) in JavaScript
+				</a>
+				on the CKEditor Blog to find out more about the presented function.
+			</p>
+			<p>
+				Please bear in mind, this mechanism only works in Chrome and it requires this flag:
+				<code>chrome://flags#enable-experimental-web-platform-features</code>
+				to be turned on.
+			</p>
+			<p>
+
+			</p>
+		</div>
+		<canvas style="position: relative;" />
+	</div>
+	</div>
+	<script type="module">
+		import detectShape from './detector.mjs';
+		const el = document.querySelector('#imageDisplay');
+		const button = document.querySelector('input[type="file"]');
+		const canvas = document.querySelector("canvas");
+		const canvasContext = canvas.getContext('2d');
+		const infoText = document.querySelector(".info-text")
+
+		button.addEventListener('click', (event) => { event.target.value = null }) // Clearing the image - needed if you want to use the same one for different shape detections.
+		button.addEventListener('change', async ({ target }) => {
+			// Getting data for detection.
+			const [image] = target.files;
+			const detectionType = document.querySelector('input[name="detection-type"]:checked').value;
+			const detected = await detectShape(image, detectionType);
+
+			// Drawing image in a <canvas>.
+			const imageData = await createImageBitmap(image);
+			drawImage(imageData);
+			
+			// Marking each detection on the image.
+			detected.forEach(element => {
+				markDetection(element);
+			});
+			infoText.style.display = "none"; // Hiding info text after loading the image.
+
+			console.log( detected ); // Detection output data.
+		});
+
+		function drawImage(img) { // Drawing loaded image on the canvas.
+			canvas.width = img.width;
+			canvas.height = img.height;
+
+			canvasContext.drawImage(img, 0, 0);
+		}
+
+		function markDetection(element) { // Function for marking the detected areas.
+			const lineWidth = element.landmarks ? 5 : 2;
+			canvasContext.strokeStyle = "lime";
+			canvasContext.lineWidth = lineWidth;
+			canvasContext.strokeRect(element.boundingBox.left, element.boundingBox.top, element.boundingBox.width, element.boundingBox.height);
+		}
+	</script>
+</body>
+
+</html>
```

Comment:
Missing newline at the end of the file.

```suggestion
</html>
```